translations/es.yaml

```diff
@@ -93,6 +93,7 @@ general:
   dashboard: Tablero
   deactivate: Desactivar
   defaultInstructors: "Instructores Predeterminados"
+  defaultVirtualLearningLink: "Enlace de Aprendizaje Virtual Predeterminado"
```

Comment:
see comment above.

Reply:
oh oops - I changed it from a different title and then it got out of alphabetization - I will fix this.